examples/common.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
Could you add some README doc for how to run these example?

Reply:
Added